requirements/testing.txt

```diff
@@ -143,8 +140,6 @@ rsa==4.9
     # via google-auth
 setuptools-git==1.2
     # via prophet
-shillelagh[gsheetsapi]==1.2.6
```

Comment:
Any ideas why this was dropped?

Reply:
will check

Reply:
I tested `pip compile-multi --no-upgrade` on master branch and it is still removing certain packages. So it's not related to this PR. 

appnope & requests is getting removed from development.txt, apsw and shillelagh[gsheetsapi] from testing.txt. It is mostly like related to @betodealmeida's most [recent PR](https://github.com/apache/superset/pull/14225).

Reply:
In this PR, I have kept package changes pertinent to bumping PyHive only.